<ui version="4.0" >
 <class>MvdImportResultsPage</class>
 <widget class="QWidget" name="MvdImportResultsPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QStackedWidget" name="stack" >
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="noResultsPage" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>331</width>
        <height>248</height>
       </rect>
      </property>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <item row="0" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="stackedInfoLabel" >
         <property name="text" >
          <string/>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" >
        <widget class="QProgressBar" name="progressBar" >
         <property name="value" >
          <number>100</number>
         </property>
         <property name="textVisible" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="resultsPage" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>331</width>
        <height>248</height>
       </rect>
      </property>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <item row="0" column="0" >
        <widget class="QTreeWidget" name="results" >
         <property name="rootIsDecorated" >
          <bool>false</bool>
         </property>
         <column>
          <property name="text" >
           <string>1</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="infoLabel" >
         <property name="text" >
          <string/>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
